### DISCUSSION TITLE
New stat: activation time

### DISCUSSION THREAD

**Discussion by argenkiwi** [monkeytypegame/monkeytype#5432] [Ideas]
I would like to suggest the addition of a stat that some of us may find useful: activation time. By activation time I refer to the amount of time that elapses between when a key is pressed (activated) and released (deactivated). We could show an average as well as the top and bottom percentiles. 

The main use case I have in mind is implementing [home row modifiers](https://precondition.github.io/home-row-mods) (or similar) using keyboard customization firmware (e.g., [QMK](https://docs.qmk.fm)) or software (e.g., [Kanata](https://github.com/jtroo/kanata)). 

Basically, if we want a key to have different behaviors when pressed than when held, it is important to figure out how long we activate a key for when we are typing. A key should be considered to be held when the activation time is higher than our slowest activation time when typing. Because we all type differently, I believe this is a very personal stat and I would love if my favorite typing practice tool could keep track of it for me.
